e2e/e2e.go

```diff
@@ -343,10 +349,26 @@ func main() { //nolint:funlen,gocyclo // Why: there are no reusable parts to ext
 				}
 			}
 
-			if err := provisionNew(ctx, deps, target); err != nil {
+			if err := provisionNew(ctx, target); err != nil {
 				//nolint:gocritic // Why: need to get exit code >0
 				log.Fatal().Err(err).Msg("Failed to create cluster")
 			}
+
+			wg.Add(1)
+
+			// Build docker sooner and out of critical path to speed things up.
+			// Docker build in devenv apps deploy . will be superfast then.
+			go func(wg *sync.WaitGroup) {
+				defer wg.Done()
+				log.Info().Msg("Starting early docker build")
+				if err := exec.CommandContext(ctx, "make", "docker-build").Run(); err != nil {
```

Comment:
If this fails, we're going to have no output. I imagine the idea is that the later `docker apps deploy .` will show the error, right?

Or should we just exit early if this fails and dump the entire output?

Reply:
My thinking here is that we don't care much here about the output, since docker build will be called again later in `devenv apps deploy .` (it will be much faster since images were already built). So we will see output anyway from there.